wordpress-mobile/WordPress-Aztec-Android
# Save and Restore Edit History on Device Rotation

### Fix
Keep edit history after device orientation change as described in https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/67.

### Test
1. Enter text.
2. Tap Undo.
3. Notice history.
4. Tap Redo.
5. Notice history.
6. Rotate device.
7. Tap Undo.
8. Notice history.
9. Tap Redo.
10. Notice history.

### Alternative
I <i>really</i> wanted to put this logic in `AztecText` to keep it bundled within the library, but I couldn't get it to work.  As it is now, we'll just have to add instructions for maintaining edit history in the README.  I'm open to alternative approaches.
